go/tasks/plugins/webapi/bigquery/query_job.go

```diff
@@ -0,0 +1,259 @@
+package bigquery
+
+import (
+	"encoding/json"
+	"strconv"
+
+	flyteIdlCore "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	pluginErrors "github.com/flyteorg/flyteplugins/go/tasks/errors"
+	"github.com/flyteorg/flytestdlib/errors"
+	"github.com/golang/protobuf/jsonpb"
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"google.golang.org/api/bigquery/v2"
+)
+
+type QueryJobConfig struct {
+	Location  string `json:"location"`
+	ProjectID string `json:"projectId"`
+
+	// AllowLargeResults: [Optional] If true and query uses legacy SQL
+	// dialect, allows the query to produce arbitrarily large result tables
+	// at a slight cost in performance. Requires destinationTable to be set.
+	// For standard SQL queries, this flag is ignored and large results are
+	// always allowed. However, you must still set destinationTable when
+	// result size exceeds the allowed maximum response size.
+	AllowLargeResults bool `json:"allowLargeResults,omitempty"`
```

Comment:
should this be input to the execution or task definition?

Reply:
ohh nevermind, i see you are using this as a json input

Reply:
do we want to model this as a protobuf (plugins) so that we can get the protobufs built automatically?